rs/canister_sandbox/src/replica_controller/sandboxed_execution_controller.rs

```diff
@@ -1169,6 +1176,10 @@ impl SandboxedExecutionController {
                     metrics
                         .sandboxed_execution_subprocess_rss
                         .observe(process_rss as f64);
+                    if process_rss > 0 {
```

Comment:
The `process_rss` includes the `memfd_rss`, which is basically included in the page delta, which I think is already accounted for by the replica. I assume that we can get away without counting the `memfd_rss`, correct?